packages/pwa-kit-react-sdk/src/ssr/universal/public-path.js

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * This file is used to dynamically set the webpack public path used by HMR via the global webpack variable
+ * __webpack_public_path__
+ * See https://webpack.js.org/guides/public-path/
+ *
+ * Previously, we hard coded the public path in our webpack config to '/mobify/bundle/development/'
+ * but we need something more dynamic to support namespaced /mobify paths.
+ */
+
+import {getBundlePathBase} from '@salesforce/pwa-kit-runtime/utils/ssr-namespace-paths'
+
+/* global __webpack_public_path__: writable */
+__webpack_public_path__ = `${getBundlePathBase()}/development/`
```

Comment:
I understand why you have this in here, but I feel like it doesn't make sense to be in the SDK as it's a "tool" of sorts that is used only for development when the SDK house code that actually gets bundled into the app bundle.

Reply:
I have moved this to pwa-kit-dev.